src/StaticWebAssetsSdk/Sdk/Sdk.StaticWebAssets.StaticAssets.ProjectSystem.props

```diff
@@ -22,15 +30,15 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DefaultItemExcludes>$(DefaultItemExcludes);**\node_modules\**;node_modules\**</DefaultItemExcludes>
     <DefaultItemExcludes>$(DefaultItemExcludes);**\jspm_packages\**;jspm_packages\**</DefaultItemExcludes>
     <DefaultItemExcludes>$(DefaultItemExcludes);**\bower_components\**;bower_components\**</DefaultItemExcludes>
-    <DefaultWebContentItemExcludes>$(DefaultWebContentItemExcludes);wwwroot\**</DefaultWebContentItemExcludes>
+    <DefaultWebContentItemExcludes>$(DefaultWebContentItemExcludes);@(StaticWebAssetSourcePath->'%(Identity)\**')</DefaultWebContentItemExcludes>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(EnableDefaultItems)' == 'true' And '$(EnableDefaultContentItems)' == 'true' ">
-    <!-- Publish everything under wwwroot, all JSON files, all config files and all Razor files -->
-    <Content Include="wwwroot\**" ExcludeFromSingleFile="true" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
+    <!-- Publish everything under @(StaticWebAssetSourcePath), all JSON files, all config files and all Razor files -->
+    <Content Include="%(StaticWebAssetSourcePath->Identity)\**" ExcludeFromSingleFile="true" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
```

Comment:
Why can't this just be `$(StaticWebAssetRootPath)\**`

Reply:
That was my previous approach. With the current approach (which doesn't work yet :/ ) I'm trying to be able to have a nuget package with `public` folder, but target app with `client` folder. So that libraries/packages could be shared (if the actual code make sense to be shared). 

Does it make sense to you? 

Unfortunatelly this style of msbuild globing doesn't work outside of a target